Description: Fix failing tests with Debian version of quilt
Author: Dr. Tobias Quathamer <toddy@debian.org>
Forwarded: not-needed

 Due to the patch "fail_on_missing", Debian's quilt will not
 return the expected output if a patch is empty. Therefore,
 we need to pass "-f" to make it apply the patch anyway.

--- a/test/altered-series.test
+++ b/test/altered-series.test
@@ -6,7 +6,7 @@
 < 02.patch
 < 03.patch
 
-$ quilt push -q 2
+$ quilt push -fq 2
 > Applying patch %{P}01.patch
 > Patch %{P}01.patch does not exist; applied empty patch
 > Applying patch %{P}02.patch
--- a/test/three.test
+++ b/test/three.test
@@ -170,7 +170,7 @@
 	> Removing patch %{P}patch1.diff
 	> No patches applied
 
-	$ quilt push -q 10
+	$ quilt push -fq 10
 	> Applying patch %{P}patch1.diff
 	> Applying patch %{P}patch2.diff
 	> Applying patch %{P}patch3.diff
--- a/test/missing.test
+++ b/test/missing.test
@@ -5,10 +5,10 @@
 	< missing2.diff
 
 	$ quilt push -qa
-+	> Patch patches/missing1.diff does not exist
-+	> Applying patch patches/missing1.diff
-+
-+	$ quilt push -qaf
+	> Patch patches/missing1.diff does not exist
+	> Applying patch patches/missing1.diff
+
+	$ quilt push -qaf
 	> Applying patch %{P}missing1.diff
 	> Patch %{P}missing1.diff does not exist; applied empty patch
 	> Applying patch %{P}missing2.diff
